# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1361059138 18000
#      Sat Feb 16 18:58:58 2013 -0500
# Node ID 1a2f1001d07dfa98c1e78b6cf2256d057dee78b6
# Parent  f837bdd535f7ed61256610d011da13a7079889e9
# Parent  9bfbe785e72665033cee4840fc51870b33471480
maint: merge away extra head on default

diff --git a/.hgtags b/.hgtags
--- a/.hgtags
+++ b/.hgtags
@@ -69,8 +69,9 @@ 4460c4fb20e6a5d3b1972fa737d4e00eb921545a
 551566201318bf615b27c60ccf9368f4844008bd release-3-6-2
 a95432e7309ca6fc776c02939264bb6d443f3525 release-3-6-3
 2e8eb9ac43a5f8cfaf0423814a312ed47cb80485 rc-3-6-4-0
 df1aceb8f0bc6b5b5062907931cc663467f57d93 ss-3-7-1
 858cbf6fc2ec1c232f5cf1d75dc344439b39a89c rc-3-6-4-1
 faefa1bea8ddae3cab170afdeab68d3d15c4e623 ss-3-7-2
 0000000000000000000000000000000000000000 ss-3-7-2
 23a7661e529ae9bfc91693618f8c314c31f695ca ss-3-7-2
+cc5a7d1233f3acea85648baeb754fc0e8f225225 rc-3-6-4-2
